### DISCUSSION TITLE
Air Gap Upgrade not Showing Latest Version

### DISCUSSION THREAD

**Discussion by mtkhayes3** [Security-Onion-Solutions/securityonion#2812] [Unsupported Versions]
Upgraded Standalone from 2.3.10 to 2.3.21 using soup 
System ran through upgrade as shown in message received
**_**"Performing upgrade from SecurityOnion 2.3.10 to Security Onion 2.3.21"
Ending with 
"Summary for _local_
Succeeded: 547 (changed=18)
Failed: 0
Total states run: **547**
Total run time: 143.891s
Applying playbook settings
Waiting for TheHive... connected!
Migrating thehive database if needed
$"**_**
Suspected it completed but when I checked VERSION if did not change 
Version of Security Onion did not change from 2.3.10 to 2.3.21 under /home/so/SecurityOnion/VERSION
So I rebooted but still the VERSION is not shown 2.3.21
Any assistance would be greatly appreciated.

**Comment by mtkhayes3** [comment]
The version of Security Onion does not update under /home/so/SecurityOnion/VERSION but I was able to confirm install worked 
when I did a more on the soup.log 
"Found that Security Onion 2.3.21 is currently installed"

**Reply by TOoSmOotH** [reply]
That is because that is the location of the installer from your original install. The best option to see what version something is on is to use the "Grid" section in soc. On the manager it is also can be found by running "cat /etc/soversion". This file only gets updated on the manager.

**Comment by mtkhayes3** [comment]
That you very much

**Reply by mtkhayes3** [reply]
This worked great thank you very much now to carry on
